docker-compose.yml

```diff
@@ -89,7 +89,7 @@ services:
       - RACK_ENV=production
 
   redis:
-    image: redis:alpine
+    image: redis:6.2-alpine
```

Comment:
Not directly related to the PR, but it's better to use the proper redis version in our local environments.

Reply:
Should we also set the version in INSTALL.md?

Reply:
Done